src/main/java/io/github/darealturtywurty/turtychemistry/block/ClayAlloyFurnaceBlock.java

```diff
@@ -32,6 +32,11 @@
 
 public final class ClayAlloyFurnaceBlock extends Block implements EntityBlock {
     private static final VoxelShape SHAPE = createShape();
+    private static final Component TRANSLATION_COMPONENT;
+
+    static {
+        TRANSLATION_COMPONENT = Component.translatable("container." + TurtyChemistry.MODID + ".clay_alloy_furnace");
+    }
```

Comment:
Why have you put this in a static block? Just inline the initialization.

Reply:
I think the main reason why this is in a static block is because at the time I was writing this on my laptop while I was in florida and it was easier to read on a small screen. I'll inline it because that does in fact make more sense